editor/src/com/talosvfx/talos/editor/addons/scene/widgets/directoryview/KeepStopReplaceDialog.java

```diff
@@ -0,0 +1,16 @@
+package com.talosvfx.talos.editor.addons.scene.widgets.directoryview;
+
+import com.badlogic.gdx.Gdx;
+import com.kotcrab.vis.ui.util.dialog.ConfirmDialogListener;
+import com.kotcrab.vis.ui.util.dialog.Dialogs;
+
+public class KeepStopReplaceDialog extends Dialogs.ConfirmDialog<Runnable> {
+    public KeepStopReplaceDialog(String title, String text, Runnable keep, Runnable stop, Runnable replace) {
+        super(title, text, new String[]{"Keep", "Stop", "Replace"}, new Runnable[] {keep, stop, replace}, new ConfirmDialogListener<Runnable>() {
```

Comment:
What is the difference between keep and stop?

Reply:
It's like with mac. Choosing Keep action will result in a file added, but with a new time to avoid collisions, and in case of stop it will not upload to file.

Reply:
ew mac

Reply:
🙂